subject: Define __STDC_UTF_{16,32}__ macros

These predefined macros indicates that our u and U chars/strings
are UTF-16 and UTF-32-encoded, respectively.


diff --git a/preprocess.c b/preprocess.c
index 2adb951..65dbb06 100644
--- a/preprocess.c
+++ b/preprocess.c
@@ -899,6 +899,8 @@ void init_macros(void) {
   define_macro("__STDC_NO_COMPLEX__", "1");
   define_macro("__STDC_NO_THREADS__", "1");
   define_macro("__STDC_NO_VLA__", "1");
+  define_macro("__STDC_UTF_16__", "1");
+  define_macro("__STDC_UTF_32__", "1");
   define_macro("__STDC_VERSION__", "201112L");
   define_macro("__STDC__", "1");
   define_macro("__USER_LABEL_PREFIX__", "");
